### PR TITLE
Update Udacity

### PR DESCRIPTION
**Pull request by fbernhart**
udacity

**Issue:** #3169 
**Alexa rank:** -

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I took the already used SVG.
I used Inkscape to fix size and center
I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG

### CODE REVIEW

File: icons/udacity.svg [hunk 1]
```diff
@@ -1 +1 @@
...
... 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Udacity icon</title><path d="M8.0104.137L0 4.6792v10.4584c-.015 4.8032 3.9152 8.7104 8.7754 8.7254 1.604.004 3.1151-.418 4.4142-1.154l6.4872-3.6862C22.238 17.7168 23.99 15.0766 24 12.0315V1.095L22.41.179 15.9066 3.81v11.3735c-.003.482-.025.836-.153 1.435-.216.9151-.521 1.5611-.574 1.6651-.511-.03-1.5461-.314-2.3861-.81-.42-.24-.914-.601-1.3641-1.021-.3-.285-.556-.615-.78-.945-.24-.345-.435-.7041-.585-1.0791-.165-.391-.284-.78-.375-1.185-.09-.421-.104-.8551-.104-1.2751L9.6004 1.05 8.0103.142zm3.6871 21.5839c-.436.179-.889.316-1.35.41-.477.094-.916.141-1.4031.141-.488 0-.978-.053-1.452-.151-.461-.092-.915-.234-1.3501-.417-.424-.179-.836-.404-1.219-.659-.9901-.691-1.5001-1.2911-1.9411-1.9361-.26-.39-.48-.795-.66-1.215a7.3287 7.3287 0 01-.416-1.3351c-.09-.465-.104-.961-.104-1.439V5.5791l6.0002-3.6211v10.0114c0 3.8061 2.819 6.9793 6.4502 7.7473-.135.165-.24.331-.391.48-.33.322-.675.624-1.064.877-.375.257-.645.465-1.065.644zm10.4054-8.4084c-.087.404-.211.803-.375 1.186-.439.9551-1.148 1.7702-1.74 2.2942-1.2161 1.101-2.7252 1.44-2.8982 1.44.362-.948.608-1.965.611-3.039V4.7241l4.5042-2.7251.003 10.0094c0 .431-.018.879-.105 1.296z"/></svg>
```
Comment (adamrusted) [inline icons/udacity.svg:1]: Spot on! 👌
Is this able to reduce to 3 decimals?

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/udacity.svg (hunk 1 above)
Comment (fbernhart) [reply icons/udacity.svg]: Fixed 🔧

**Review by adamrusted — CHANGES_REQUESTED** [review]
Looks like there's a few differences between PR (red) and source (black) - any chance of a rebuild? 😅
image

**Comment by fbernhart** [comment]
@adamrusted Fixed. I've as well changed the color to `#02B3E4`, which is the logo color in the original SVG file.

udacity

**Review by adamrusted — APPROVED** [review]
Perfect! 💯
Thanks @fbernhart for your contribution!

**Pull request merged**